Registry/multiplayersample.setreg

```diff
@@ -1,7 +1,15 @@
 {
     "MultiplayerSample": {
         "Settings": {
-            "WinningCoinCount": 3
+            "WinningCoinCount": 3,
+            "EnergyBall": {
+                "KnockbackDistance": 2.0,
```

Comment:
"KnockbackDistance**Meters**"? (tbh not sure of our scale, assuming)

Reply:
Done